app/models/liquid_partial_observer.rb

```diff
@@ -0,0 +1,10 @@
+class LiquidPartialObserver < ActiveRecord::Observer
+  def after_save(record)
+    LiquidRenderer::Cache.invalidate
```

Comment:
It would be cool to add a spec that shows that it doesn't fire when something else is updated